src/api/router.go

```diff
@@ -34,26 +32,9 @@ func (api *Api) ApiRouter() *gin.Engine {
 	})
 
 	if api.Config.FeatureEnabled("account") {
-		a := &auth.AccountApi{Config: api.Config, CraneDockerClient: api.Client}
-		if api.Config.AccountTokenStore == "default" {
-			a.TokenStore = token_store.NewDefaultStore()
-		} else if api.Config.AccountTokenStore == "cookie_store" {
-			a.TokenStore = token_store.NewCookieStore()
-		}
-
-		if api.Config.AccountAuthenticator == "default" {
-			a.Authenticator = authenticators.NewDefaultAuthenticator()
-		} else if api.Config.AccountAuthenticator == "db" {
-			a.Authenticator = authenticators.NewDBAuthenticator()
-		}
-
-		Authorization = chains.Authorization(a)
-		AuthorizeServiceAccess = chains.AuthorizeServiceAccess(a)
```

Comment:
this and above statement override default behaviour of Authorization, AuthorizeServiceAccess base on logic in auth module, what is the plan of giving a way of doing this?